test/data/cmumps2fhir-patient7.shex

```diff
@@ -0,0 +1,587 @@
+﻿PREFIX fhir:<http://hl7.org/fhir/> 
```

Comment:
If you originally got it from the FHIR website then I think it was actually generated by the FHIR spec build process, which is what we want.  I just wanted to verify that you were starting from that one.  Thanks.

BTW, did you file a shex issue about the fact that the shex materialize tool fails when given a file with embedded comments?  Please do if you haven't already.